README.md

```diff
@@ -3,6 +3,8 @@
 This repository includes backup and recovery utilities for
 [GitHub Enterprise Server][1].
 
+**UPDATE**: `backup-utils` now requires [awk](https://www.gnu.org/software/gawk) and [moreutils](https://joeyh.name/code/moreutils).
```

Comment:
There are several variants of awk so we should be explicit here.

```suggestion
**UPDATE**: `backup-utils` now requires [GNU awk](https://www.gnu.org/software/gawk) and [moreutils](https://joeyh.name/code/moreutils).
```

Reply:
Done